helloworld-samples/function-http-nodejs/main.js

```diff
@@ -0,0 +1,13 @@
+module.exports.main = main = async (args) => {
+    const url = 'https://httpbin.org/get';
```

Comment:
We could also extend this sample and conditionally use the users POST payload and send it to httbin.org/post, which will echo it. 
What do you think?

Reply:
good idea added 👍